src/QuestionRadioItem.tsx

```diff
@@ -70,32 +70,44 @@ export interface Props {
   onChange?: (string) => void;
 }
 
-@inject((store: any) => ({
-  isPreview: store.model.isPreview,
-}))
-@observer
-export default class QuestionRadioItem extends React.Component<Props> {
+class QuestionRadioItem extends React.Component<QuestionRadioItemProps> {
   handlePress = () => {
     const checked = !this.props.checked;
     if (checked && this.props.onChange) {
       this.props.onChange(this.props.value);
     }
-  }
+  };
 
   render() {
     const { isPreview, label, text, value, checked } = this.props;
     return (
-      <TouchableWithFeedback style={[styles.container, checked && styles.containerChecked]} onPress={this.handlePress}>
-        {label &&
+      <TouchableWithFeedback
+        style={[styles.container, checked && styles.containerChecked]}
+        onPress={this.handlePress}
+      >
+        {label && (
           <View style={[styles.label, checked && styles.labelChecked]}>
             <Text style={styles.labelText}>{label}</Text>
           </View>
-        }
-        <Text style={[styles.text, checked && styles.textChecked]}>{text || value}</Text>
-        {!isPreview &&
-          <Image style={styles.radio} source={checked ? require('./images/radio-checked.png') : require('./images/check.png')} />
-        }
+        )}
+        <Text style={[styles.text, checked && styles.textChecked]}>
+          {text || value}
+        </Text>
+        {!isPreview && (
+          <Image
+            style={styles.radio}
+            source={
+              checked
+                ? require('./images/radio-checked.png')
+                : require('./images/check.png')
+            }
+          />
+        )}
       </TouchableWithFeedback>
     );
   }
 }
+
+export default inject((store: any) => ({
+  isPreview: store.model.isPreview,
+}))(observer(QuestionRadioItem));
```

Comment:
This is the recommended approach (not using decorators) which fixed the rendering issue.